d2ir/pattern.go

```diff
@@ -0,0 +1,59 @@
+package d2ir
+
+import (
+	"strings"
+
+	"oss.terrastruct.com/d2/d2graph"
+)
+
+func (m *Map) doubleGlob(pattern []string) ([]*Field, bool) {
+	if !(len(pattern) == 3 && pattern[0] == "*" && pattern[1] == "" && pattern[2] == "*") {
+		return nil, false
+	}
+	var fa []*Field
+	m._doubleGlob(&fa)
+	return fa, true
+}
+
+func (m *Map) _doubleGlob(fa *[]*Field) {
+	for _, f := range m.Fields {
+		if _, ok := d2graph.ReservedKeywords[f.Name]; ok {
+			if _, ok := d2graph.BoardKeywords[f.Name]; !ok {
```

Comment:
it shouldn't go into `layers` right? that's completely new board

Reply:
It should, you might be setting all shapes a `page` everywhere. Or some kind of default styling.